src/main/java/org/hyperledger/aries/webhook/EventHandler.java

```diff
@@ -35,92 +35,94 @@ public void handleEvent(String eventType, String json) {
 
         try {
             if ("connections".equals(eventType)) {
-                parser.parseValueSave(json, ConnectionRecord.class).ifPresent(this::handleConnection);
+                handleConnection(parser.parseValueSave(json, ConnectionRecord.class).get());
```

Comment:
All the changes in this class are totally unnecessary, also they break backwards compatibility with existing clients. Basically what you intent with your changes is allowing null payloads in the EventParser. But why is that needed at all? As I understand you in the websocket it can happen that there is a Ping event without a payload and you still want to notify implementing clients that there was such an event?  In my opinion an event without payload is not very useful, but if it really is needed then the EventParser should return an empty object in that case. You could very easily achieve this by simply passing "{}" to the handler instead of null in your websocket handler implementation without making any changes here. As empty payloads are only happening with websockets this is probably the better approach.